tests/Integration/Command/FindPointsTest.php

```diff
@@ -94,7 +94,7 @@ private function getCommand(): FindPoints
         return new FindPoints($this->client, new Serializer());
     }
 
-    private function getSuccessfulBody(): string
+    private function getSuccessfulBody(): string|false
```

Comment:
tak nie róbmy. Jak oczekujemy string'a to powinien być string. jak musisz to rzuć w metodzie wyjątkiem jak `bool`

Reply:
Done :)